### PR TITLE
cleanup template handling of peers

### PR DESCRIPTION
**Pull request by Phil-Friderici**

### CODE REVIEW

**Comment by Phil-Friderici** [comment]
@ghoneycutt now it's complete
keeping fingers crossed that this can be merged fluent after https://github.com/ghoneycutt/puppet-module-ntp/pull/65 was merged

File: templates/ntp.conf.erb [hunk 1]
```diff
@@ -36,12 +36,12 @@ server <%= server %><% if @server_options != 'UNSET' %> <%= @server_options %><%
 
 <% if @my_peers != ['UNSET'] -%>
 # Peers
-<%   if @peers_type == 'string' or @peers_type == 'array' -%>
+<%   if @my_peers.is_a?(Array) -%>
```
Comment (ghoneycutt) [inline templates/ntp.conf.erb]: what if my_peers is a string?

**Comment by ghoneycutt** [comment]
I rebased your commit in PR #68 so that the ordering of commits was correct and only this new commit would show up.

File: templates/ntp.conf.erb (hunk 1 above)
Comment (Phil-Friderici) [inline templates/ntp.conf.erb]: not possible for this output variable, please see the input handling here: https://github.com/ghoneycutt/puppet-module-ntp/blob/master/manifests/init.pp#L69-L85

Comment (ghoneycutt) [inline templates/ntp.conf.erb]: Aha!

**Comment by Phil-Friderici** [comment]
thanks for merging and sorry for the extra mile

**Pull request closed without merging**